### ISSUE TITLE
[MAS2.4.7] [Keyboard Navigation-Create a new bot configuration] Visual focus indicator is not clearly visible for Learn more links 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1742]
**Actual Result:**
visual focus indicator is not clearly visible for Learn more links present in the new bot configuration 

**Expected Result:**
visual focus indicator should be clearly visible for Learn more links

**Note:** Same issue repro for below path 
Bot Explorer->Endpoint->+symbol:
visual focus indicator is not clearly visible for Learn more links present in the Add an endpoint for your bot 
Please refer the attachment:MAS2.4.7_Choose.a.different.file.location.zip

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the Welcome tab screen
3.Navigate to the create a new bot configuration  and select it 
4.Dialog gest opened and try to navigate to the Learn more links using tab key 
5.Observe the issue

**User Impact:**
users who rely on Keyboard navigation will be affected the keyboard focus indicator is not clearly visible on control so that keyboard dependant users will be affected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}](url)
[MAS2.4.7_Create new bot configuration.zip](https://github.com/microsoft/BotFramework-Emulator/files/3515346/MAS2.4.7_Create.new.bot.configuration.zip)
[MAS2.4.7_Choose a different file location.zip](https://github.com/microsoft/BotFramework-Emulator/files/3559083/MAS2.4.7_Choose.a.different.file.location.zip)

**Comment by Kumar2608** [comment]
Verified in latest bot emulator Version 4.5.2-78641 and issue got fixed Hence closing this bug
Now VFI is clearly visible on learn more links
